README.md

```diff
@@ -94,19 +90,18 @@ I feel like learning about Wayland is a better use of my effort.
 Lack of EWMH means some applications (especially games) will break when 
 trying to go fullscreen.
 
-Authors
-=======
-
-See [the credits](CREDITS.md) for a full list of everybody who's helped.
-
-9wm is maintained by Jacob Adams <tookmund@gmail.com>.
 
-9wm was maintained by Neale Pickett <neale@woozle.org>.
+## Authors
 
-9wm was written by David Hogan (dhog@cs.su.oz.au), a postgraduate
-student at the Basser Department of Computer Science, University
-of Sydney <http://unauthorised.org/dhog/index.html>
-David Hogan died in 2003.
+See the [CREDITS.md](CREDITS.md) for a full list of everybody who's helped.
 
+* 9wm is maintained by Jacob Adams <tookmund@gmail.com>
+* 9wm was maintained by Neale Pickett <neale@woozle.org>
 
+9wm was originally written by David Hogan <dhog@cs.su.oz.au>, a postgraduate student at the [Basser Department of Computer Science, University of Sydney](http://unauthorised.org/dhog/index.html). He passed away in 2003.
```

Comment:
dhog's homepage should not be the link opened when someone clicks on "Basser Department of Computer Science".